api/v1/flow/flow.proto

```diff
@@ -122,21 +122,38 @@ enum TraceObservationPoint {
     // observation point is always going to be present on trace events.
     UNKNOWN_POINT = 0;
 
+    // TO_PROXY indicates network packets are to be redirected to l7 proxy.
     TO_PROXY = 1;
+    // TO_HOST indicates network packets are received from host namespace.
```

Comment:
I think this should be `are sent to the host interface/namespace` or `are received by the host namespace`. This is used when a packet is redirected to the `cilium_host` interface.

Reply:
> I think this should be are sent to the host interface/namespace or are received by the host namespace

Agreed.

> This is used when a packet is redirected to the cilium_host interface.
Please correct me if I am wrong.

From current code, it seems `TO_HOST` are for packets to host endpoints, and from the endpoint bpf map, flows to eth0 will also be traced as TO_HOST
```
root@aks-nodepool1-25490788-vmss000003:/home/cilium# cilium bpf endpoint list
IP ADDRESS     LOCAL ENDPOINT INFO
10.0.0.199:0   (localhost) // cilium_host
10.0.0.231:0   id=3148  flags=0x0000 ifindex=17  mac=4A:CC:C0:AC:6C:33 nodemac=EE:81:0C:44:03:2D
10.0.0.220:0   id=33    flags=0x0000 ifindex=21  mac=E6:A3:CD:E8:A7:86 nodemac=56:AD:5E:3D:82:E9
10.0.0.67:0    id=478   flags=0x0000 ifindex=19  mac=2A:BB:15:10:4A:34 nodemac=82:16:20:4B:6A:CC
10.0.0.176:0   id=219   flags=0x0000 ifindex=15  mac=72:9F:24:B8:D5:9E nodemac=56:D2:1B:4E:63:70
10.224.0.4:0   (localhost) // eth0
10.0.0.200:0   id=1545  flags=0x0000 ifindex=25  mac=52:1F:14:B9:07:A2 nodemac=BA:40:E6:F7:28:91
10.0.0.162:0   id=3677  flags=0x0000 ifindex=13  mac=FA:05:87:D1:B0:65 nodemac=2E:A5:3C:9B:43:AC
```
https://github.com/cilium/cilium/blob/master/bpf/bpf_lxc.c#L979

Reply:
> From current code, it seems `TO_HOST` are for packets to host endpoints, and from the endpoint bpf map, flows to eth0 will also be traced as TO_HOST

They shouldn't be. AFAIK, we don't rely on this map to decide if we should be doing a TO_HOST or a TO_NETWORK. We rely on the attach points and carry that information (via the `from_host` boolean) across the code.